c7n/tags.py

```diff
@@ -149,30 +149,30 @@ class TagTrim(Action):
 
     .. code-block :: yaml
 
-      - policies:
+       policies:
          - name: ec2-tag-trim
            comment: |
              Any instances with 48 or more tags get tags removed until
              they match the target tag count, in this case 47 so we
              that we free up a tag slot for another usage.
            resource: ec2
            filters:
-               # Filter down to resources which already have 8 tags
-               # as we need space for 3 more, this also ensures that
-               # metrics reporting is correct for the policy.
-               type: value
-               key: "[length(Tags)][0]"
-               op: ge
-               value: 48
+                 # Filter down to resources which already have 8 tags
+                 # as we need space for 3 more, this also ensures that
+                 # metrics reporting is correct for the policy.
+               - type: value
+                 key: "[length(Tags)][0]"
```

Comment:
another spot the array could be dropped for just `length(Tags)` there's a few more in this pr on lines modified.